runtime/gc_glue_java/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -700,13 +700,20 @@ MM_CollectorLanguageInterfaceImpl::scavenger_switchConcurrentForThread(MM_Enviro
 			j9tty_printf(PORTLIB, "%p: Nursery [%p,%p] Evacuate [%p,%p] GS [%p,%p] Section size 0x%zx, sections %lu bit offset %lu bit mask 0x%zx\n",
 					vmThread, nurseryBase, nurseryTop, gsBase, gsTop, pageBase, pageTop, _extensions->getConcurrentScavengerPageSectionSize() , sectionCount, startOffsetInBits, bitMask);
 		}
+#if defined(J9VM_GC_COMPRESSED_POINTERS) && defined(J9VM_ARCH_X86)
```

Comment:
I'd like to see a GC reviewer on this bit.

Reply:
@amicic Could you take a look?

Reply:
In particular, would it make sense to have some `#error` in here when CS is defined, but the platforms options don't have a good answer for the values (and some comments).